routers/api/v1/api.go

```diff
@@ -43,6 +43,7 @@
 //          type: apiKey
 //          name: Authorization
 //          in: header
+//          description: API tokens must be prepended with "token ".
```

Comment:
```suggestion
//          description: API tokens must be prepended with "token".
```

Reply:
The space is important - this way it's clear that it has to have "token" and a space there.

Reply:
How about:

```suggestion
//          description: API tokens must be prepended with "token" followed by a space.
```